minesweeper.py

```diff
@@ -12,6 +12,9 @@
 SIZE_X = 10
 SIZE_Y = 10
 
+FIXED_MINE_NUMBER = True #if True set a fixed number of mines, rather than a random outcome
```

Comment:
might be nice to add argument parser and set this to `False` if run with `--random` 🤔